### ISSUE TITLE
Migrate/merge with Vox Pupuli module?

### ISSUE THREAD

**Issue by bastelfreak** [xaque208/puppet-python#49]
Hi!
the original source of this module,stankevich/puppet-python, got migrated to Vox Pupuli. Are you interested in merging both modules since Vox Pupuli maintains it now properly, or archive your version?

Cheers, Tim

**Comment by xaque208** [comment]
Yes, I'd like to not maintain this module any more, but I don't have much time to work on merging.  I'm happy enough to give it up.  I removed support for lots of items I didn't feel like testing.  Proxy support iirc, and a couple other non-standard python deployments that some user might find useful have been removed, but I didn't think it was worth the complexity.  I'll at least update the changelog, and see what you think.

**Comment by rick-pri** [comment]
The voxpupuli module doesn't currently have FreeBSD support which we rely on this module for.  On the other hand we're moving away from FreeBSD, slowly, and are finding issues in this module's Ubuntu support so are caught in a bind where we need support for both platforms.  Making this module work with Ubuntu seems to be easier path for us for now.  

I'm currently fixing up the `--no-use-wheel` issues that I'm now coming across now that we can try to use the venv functionality.  Unfortunately, years later, I'm still not particularly proficient with Ruby to understand what is going on in this module in places.
